docs/monoDrive_home/CarSim.md

```diff
@@ -0,0 +1,95 @@
+# CarSim
+
+By default, the monoDrive Simulator uses PhysX for vehicle dynamics. Alternatively, monoDrive also supports co-simulation with CarSim.
+
+## Prerequisites
+
+- Windows 10
+- monoDrive Scenario Editor
+- monoDrive CarSim integration enabled
+    - Open `VehicleAI.uproject` with a text editor
+    - Enable the two required plugins
+    - Launch the editor and the integration will be built
```

Comment:
this bullet should be below the json

Reply:
![Screen Shot 2020-07-13 at 8 48 15 AM](https://user-images.githubusercontent.com/41026694/87312107-c0d59880-c4e5-11ea-9fc5-ef587c1345d6.png)
 For some reason, the markdown for the json forces the next line to be a separate bullet point instead of a sub-bullet.

Reply:
ah okay that's annoying, maybe we could just make this a full sub-section of the pre-reqs? this is the most involved anyways

#### monoDrive CarSim integration enabled
- step
- step
```
code
```
- step

Reply:
Good idea, added sub headings for the other sections as well to balance out the section: 
![Screen Shot 2020-07-13 at 9 08 36 AM](https://user-images.githubusercontent.com/41026694/87314246-920cf180-c4e8-11ea-8fd2-8d7500ea3149.png)